### PR TITLE
Fix mail

### PR DESCRIPTION
**Pull request by RoyTakanen**
- Add DMARC record
- Use SPF hardfail
- Move stuff to `Mailerlite`-function

### CODE REVIEW

File: dns/dnsconfig.js [hunk 1]
```diff
@@ -52,13 +52,14 @@ D('testausserveri.fi', REG_NONE, NO_PURGE, DnsProvider(DNS_CLOUDFLARE),
...
     Google(),
-    Mailerlite(),
-    TXT('@', 'v=spf1 include:_spf.mlsend.com include:_spf.google.com ~all')
+    Mailerlite()
 );
 
 function Mailerlite() {
     return [
         TXT('ml._domainkey', 'k=rsa; p=MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDZPVgEYMyfzZ2ybrXBQRS7uifdpzB0SNMoWMCYnsX46vU3KZ71Iz6tgcQGk4DOhRkAP5iLxNDm/s5SxD6Esn3rFHd2Cu5yIwCDLYBidyqoaa1QWmmglkFkglJXvQBp5XVX5ZXunTUHf2Sqy3MMQU9/5rY4xpRRaLVs8Yvt6i9Y7QIDAQAB'),
+        TXT('_dmarc', 'v=DMARC1; p=reject; rua=mailto:dmarc@testausserveri.fi; adkim=s; aspf=s;'),
```
Comment (ahnl) [inline dns/dnsconfig.js:61]: General DMARC record doesn't seem to be related in any way to [Mailerlite](https://www.mailerlite.com). I think we should move it out of this function.

**Review by ahnl — CHANGES_REQUESTED** [review]

File: dns/dnsconfig.js (hunk 1 above)
Comment (ahnl) [reply dns/dnsconfig.js:61]: Actually neither do the other ones seem to belong here. 🤔

Comment (RoyTakanen) [reply dns/dnsconfig.js:61]: Oh. I thought that Mailerlite is supposed to contain all mail related records :P. I will move DMARC out of it.

Comment (RoyTakanen) [reply dns/dnsconfig.js]: Now it should be fixed.

Comment (RoyTakanen) [reply dns/dnsconfig.js]: > Actually neither do the other ones seem to belong here. thinking

Should I move the SPF also out from the function?

Comment (ahnl) [reply dns/dnsconfig.js]: Yep, SPF too.

Comment (RoyTakanen) [reply dns/dnsconfig.js]: Done.

**Review by ahnl — APPROVED** [review]
